src/main/java/com/jalasoft/sdfc/pages/Signin.java

```diff
@@ -0,0 +1,30 @@
+package com.jalasoft.sdfc.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class Signin extends AbstractPage {
+
+    @FindBy(css = "#username")
+    private WebElement userNameTextField;
+
+    @FindBy(css = "#password")
+    private WebElement passwordTextField;
+
+    @FindBy(css = "#Login")
+    private WebElement loginButton;
+
+    public Signin() {
+        driver.get("https://login.salesforce.com/");
+    }
+
+    public Header loginAs(String userName, String password) {
+        action.setValue(userNameTextField, userName);
+        action.setValue(passwordTextField, password);
+        action.click(loginButton);
+
+        wait.until( ExpectedConditions.titleIs("Verify Your Identity | Salesforce"));
```

Comment:
I think this is not necessary anymore, remove it if this is the case.

Reply:
delete just line 27 I think wait.until( ExpectedConditions.titleIs("Verify Your Identity | Salesforce"));

Reply:
Hi, 
Line 27, "Verify Your Identity | Salesforce", was removed from Signin.java file